python/xorbits/pandas/mars_adapters/core.py

```diff
@@ -203,3 +200,27 @@ def wrapped(*args, **kwargs):
             return from_mars(ret)
 
     return attach_cls_member_docstring(wrapped, member_name, data_type=data_type)
+
+
+def wrap_iteration_functions(
+    member_method: Callable,
+    member_name: str,
+    attach_docstring: bool,
+):
+    """
+    Wrapper for iteration functions.
+    """
+
+    @functools.wraps(member_method)
+    def wrapped(self: MarsEntity, *args, **kwargs):
+        if own_data(self):
+            # if own data, return iteration on data directly
+            return getattr(self.op.data, member_name)(*to_mars(args), **to_mars(kwargs))
+        else:
+            return from_mars(member_method(self, *to_mars(args), **to_mars(kwargs)))
+
+    if attach_docstring:
+        return attach_cls_member_docstring(wrapped, member_name)
```

Comment:
The docstring attaching seemed to be failed. I checked the docstring of `iterrows` and the docstring was emtpy.

Reply:
Fixed.